### DISCUSSION TITLE
Documentation for IDE suppot

### DISCUSSION THREAD

**Discussion by Chris2011** [community/community#17272] [Copilot Conversations]
As I understand and see it correctly, the code pilot code is not open source, right?

I wanted to had a look into the source or into the documentation to understand how I can implement this tool as a plugin into Apache NetBeans or any other IDE that wants this. So can you please point me to the correct documentation for implementing into a new IDE? I mean what do I need to send, which package do I need to use for example is there a Java wrapper? etc.

**Comment by vityakut** [comment]
No

**Comment by jfullstackdev** [comment]
how about JetBrains ? 

https://docs.github.com/en/copilot/getting-started-with-github-copilot/getting-started-with-github-copilot-in-a-jetbrains-ide

**Reply by Chris2011** [reply]
What do you mean exactly?
